Gemfile

```diff
@@ -7,115 +7,206 @@ end
-gem "sprockets", "~> 4.0.2"
-gem "toastr-rails"
-gem "uglifier", ">= 1.3.0"
-gem 'webpacker', '~> 5.4'
-gem "yajl-ruby"
-gem "recaptcha"
-gem "paper_trail" # for tracking history of InventoryItem
+gem "image_processing"
+gem "sprockets", "~> 4.0.0"
 
 group :production do
-  gem 'lograge' # Reduce the noise of logs and include custom fields to it for easier access
+  # Reduce the noise of logs and include custom fields to it for easier access
+  gem 'lograge'
+  # Profiler (third party app) showing performance and metrics.
   gem "skylight"
 end
 
 group :development, :test, :staging do
+  # Generate models based on factory definitions.
   gem 'factory_bot_rails'
+  # Ensure the database is in a clean state on every test.
   gem "database_cleaner", '1.8.5'
+  # Generate fake data for use in tests.
   gem 'faker'
 end
 
 group :development, :test do
+  # Command line tool for better object printing.
   gem "awesome_print"
+  # Scan for vulnerabilities and other static analysis.
   gem "brakeman"
-  gem "fakeredis", require: "fakeredis/rspec"
+  # Use a fake in-memory Redis instance in tests.
+  gem "fakeredis", require: "fakeredis/rspec" # NEED?
+  # Re-run specs when files change.
   gem "guard-rspec"
+  # Split tests in a suite across multiple runners.
   gem "knapsack_pro"
+  # Show core documentation in command line.
   gem "pry-doc"
+  # Rails plugin for command line.
   gem "pry-rails"
+  # Connect to a remotely running command line instance.
   gem "pry-remote"
+  # Add-on for command line to create a simple debugger.
   gem "pry-nav"
-  gem "rb-readline", "~> 0.5.3"
-  gem "rspec-rails", "~> 5.0.2"
+  gem "rb-readline", "~> 0.5.0" # NEED?
+  # RSpec behavioral testing framework for Rails.
+  gem "rspec-rails", "~> 5.0.0"
+  # Static analysis / linter.
   gem "rubocop"
-  gem "rubocop-rails", "~> 2.9.1"
+  # Rails add-on for static analysis.
+  gem "rubocop-rails", "~> 2.9.0"
+  # Default rules for Rubocop.
   gem "standard", "~> 1.0"
-  gem "terminal-notifier-guard"
-  gem "terminal-notifier"
 end
 
 group :development do
+  # Show database columns and indexes inside files.
   gem "annotate"
+  # Used as a dependency for better_errors.
   gem "binding_of_caller"
+  # Show a better error page on development, including a REPL.
   gem "better_errors"
+  # Detect N+1 and other slow queries.
   gem "bullet"
+  # Run multiple processes from a Procfile (web, jobs, etc.)
   gem 'foreman'
+  # Open sent e-mails in the browser instead of trying to send to a real mail server.
   gem "letter_opener"
-  gem "listen", "~> 3.7.1"
+  # Used as a dependency for Guard.
+  gem "listen", "~> 3.7.0"
+  # Generate a diagram based on Rails models.
   gem "rails-erd"
+  # Allows to create a console in the browser.
   gem "web-console"
 end
 
 group :test do
-  gem "capybara", "~> 3.36"
+  # Test using browsers.
+  gem "capybara", "~> 3.0"
+  # Create screenshots when doing browser tests.
   gem "capybara-screenshot"
-  gem "launchy"
+  gem "launchy" # NEED?
```

Comment:
Same -- just delete?

Reply:
I wasn't sure if there was some development process that relied on it (I couldn't find reference in the code). I'm good to delete.